src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectProtocolValidator.cs

```diff
@@ -334,6 +326,38 @@ public virtual void ValidateUserInfoResponse(OpenIdConnectProtocolValidationCont
         }
 
         /// <summary>
+        /// Calls <see cref="Dispose(bool)"/> and <see cref="GC.SuppressFinalize"/>
+        /// </summary>
+        public void Dispose()
+        {
+            Dispose(true);
+            GC.SuppressFinalize(this);
+        }
+
+        /// <summary>
+        /// Calls <see cref="HashAlgorithm.Dispose()"/> to release this managed resources.
```

Comment:
"to release *the* managed resources". I noticed we have this typo in other places too.